libraries/freertos_plus/standard/https/include/iot_https_client.h

```diff
@@ -284,6 +300,55 @@ IotHttpsReturnCode_t IotHttpsClient_InitializeRequest(IotHttpsRequestHandle_t *
 IotHttpsReturnCode_t IotHttpsClient_AddHeader(IotHttpsRequestHandle_t reqHandle, char * pName, char * pValue, uint32_t len);
 /* @[declare_https_client_addheader] */
 
+/**
+ * @brief Writes request body to the network for the request represented by reqHandle.
+ * 
+ * This function is intended to be used by an asynchronous request. It must be called within the 
+ * #IotHttpsClientCallbacks_t.writeCallback.
+ * 
+ * In HTTP/1.1 the headers are sent on the network first before any body can be sent. The auto-generated header
+ * Content-Length is taken from the len parameter and sent first before the data in parameter pBuf is sent. In order for this function
+ * to be called twice with variable lengths of data, a request with Transfer-Encoding: chunked would be needed. This 
+ * library does not support Transfer-Encoding: chunked requests, so this function cannot be called more than once in 
+ * #IotHttpsClientCallbacks_t.writeCallback for an HTTP/1.1 request. 
+ * 
+ * For placeholder possible future support of sending a variable legnth body, isComplete is used so that this 
+ * function can be called more than once.
```

Comment:
Can we say that this inComplete is ignored currently, or that this should be set to 1 always in current version of the library?

Reply:
Ok